tests/snapd

```diff
@@ -59,20 +58,12 @@ cleanup() {
 }
 trap cleanup EXIT HUP INT TERM
 
-# Configure to use the proxy
-curl -s http://canonical-lxd.stgraber.org/config/snapd.sh | sh
-
-if [ "${track}" = "latest" ]; then
-    snapd_channel=${channel}
-else
-    snapd_channel=${track}/${channel}
-fi
-
-echo "==> Installing LXD snap from ${track}/${channel}"
+echo "==> Installing LXD snap from ${LXD_SNAP_CHANNEL}"
 snap remove lxd || true
 snap version
 snap list
-snap install lxd --channel="${snapd_channel}" --cohort=+
+snap install lxd --channel="${LXD_SNAP_CHANNEL}" --cohort=+
+snap set lxd daemon.preseed='{"config": {"core.https_address": "[::]:8443"}}'
```

Comment:
If the intention is to have a network listener, `lxd init --auto --network-address "[::]" --network-port "8443"` should be more straight forward.